File: C/unit_2/Ff.c
#include <stdio.h>
#include <math.h>
int main()
{
  float wcf, v, t;
  printf("The Tempreature of the Surrounding : \n");
  scanf("%f", &t);
  printf("The Wind Velocity of the Surrounding : \n");
  scanf("%f", &v);
  wcf = 35.74 + .6215 * t + (.4275 * t - 35.75) * pow(v, .16);
  printf("The Wind Chill Factor of the Surrounding is : %f", wcf);
  return 0;
}

File: Java/unit_2/Ff.java
package Java.unit_2;

import java.util.Scanner;

public class Ff {
    public static void main(String[] args) {
        double wcf, v, t;
        Scanner sc = new Scanner(System.in);
        System.out.println("The Tempreature of Surrounding : \n");
        t = sc.nextDouble();
        System.out.println("The Wind Velocity of Surrounding : \n");
        v = sc.nextDouble();
        wcf = 35.74 + .6215 * t + (.4275 * t - 35.75) * Math.pow(v, 16);
        System.out.println("The Wind Chill Factor of the Surrounding is : " + wcf);
        sc.close();
    }
}
